### PR TITLE
SAAS-542 refactoring e2e tests code

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,16 @@ When developing locally, you'll want to route your API requests to a dev server.
...
+### Running E2E tests locally
+
+To execute E2E tests locally you need to set related environment variables first. By default `.env.local` file is used as a source,
+so you have to create it from `.env`:
+
+```shell script
+cp .env .env.local
+```
+
+Then we should update `.env.local` with correct secrets.
```
Comment (nicolalamacchia) [inline README.md:29]:
```suggestion
Then you should update `.env.local` with correct secrets.
```

File: README.md [hunk 2]
```diff
@@ -16,3 +16,16 @@ When developing locally, you'll want to route your API requests to a dev server.
 ```json
 "proxy": "https://platform-dev.percona.com"
 ```
+
+### Running E2E tests locally
+
+To execute E2E tests locally you need to set related environment variables first. By default `.env.local` file is used as a source,
+so you have to create it from `.env`:
+
+```shell script
```
Comment (nicolalamacchia) [inline README.md:25]:
```suggestion
```sh
```

or `shell-script`

File: README.md (hunk 1 above)
Comment (yurkovychv) [reply README.md]: done, thanks

File: README.md (hunk 2 above)
Comment (yurkovychv) [reply README.md]: done, thanks

**Review by puneet0191 — DISMISSED** [review]

**Review by nicolalamacchia — DISMISSED** [review]

**Review by atymchuk — APPROVED** [review]

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**